src/intrinsics.cpp

```diff
@@ -918,6 +920,180 @@ static Value *emit_intrinsic(intrinsic f, jl_value_t **args, size_t nargs,
     Value *den;
     Value *typemin;
     switch (f) {
+    HANDLE(bytevec_ref,2) {
```

Comment:
In case it's not obvious, this code is not meant to be in production-quality state – this is a request for debugging help.